### ISSUE TITLE
Sync suddenly fails on all plattforms

### ISSUE THREAD

**Issue by lyrixderaven** [bitwarden/core#243]
After running a self-hosted dockerized instance (following the official instructions and scripts)  for a few months, suddenly syncing fails on the chrome/firefox extensions. The iOS app also crashes when trying to sync.

I've tried using fresh chrome/firefox profiles, reinstalled the clients, created a new account on the server and tried exporting / purging / importing the credentials again - to no avail.

I've also ensured the server is up to date (via `./bitwarden.sh updateself` and `./bitwarden.sh update` as well as, just for good measure, `./bitwarden.sh updatedb`) - no change. The docker containers tell me the versions are 

```
bitwarden/identity:1.16.0   
bitwarden/icons:1.16.0      
bitwarden/nginx:1.16.0      
bitwarden/api:1.16.0        
bitwarden/web:1.22.0        
bitwarden/mssql:1.16.0      
bitwarden/attachments:1.16.0
```

Weirdly, when I create a new account and sync against that empty account, it works. If I add even a single new credential, syncing fails. 

Any suggestions? I'd be willing to look into the issue myself, if anyone could give me some pointers towards enabling better debug logs on the server.

**Comment by lyrixderaven** [comment]
Then maybe the updater is broken? Here's my `./bitwarden.sh update` output:

```
 _     _ _                         _
| |__ (_) |___      ____ _ _ __ __| | ___ _ __
| '_ \| | __\ \ /\ / / _` | '__/ _` |/ _ \ '_ \
| |_) | | |_ \ V  V / (_| | | | (_| |  __/ | | |
|_.__/|_|\__| \_/\_/ \__,_|_|  \__,_|\___|_| |_|

Open source password management solutions
Copyright 2015-2018, 8bit Solutions LLC
https://bitwarden.com, https://github.com/bitwarden

===================================================

Docker version 17.05.0-ce, build 89658be
docker-compose version 1.18.0, build 8dd22a9

Stopping bitwarden-identity    ... done
Stopping bitwarden-icons       ... done
Stopping bitwarden-nginx       ... done
Stopping bitwarden-api         ... done
Stopping bitwarden-web         ... done
Stopping bitwarden-mssql       ... done
Stopping bitwarden-attachments ... done
Removing bitwarden-identity    ... done
Removing bitwarden-icons       ... done
Removing bitwarden-nginx       ... done
Removing bitwarden-api         ... done
Removing bitwarden-web         ... done
Removing bitwarden-mssql       ... done
Removing bitwarden-attachments ... done
Removing network docker_default
1.17.2: Pulling from bitwarden/setup
Digest: sha256:cf55e0288b6392ee9c35022089396c3c561773ddc963945f1afea8fce359757f
Status: Image is up to date for bitwarden/setup:1.17.2
Building docker environment files.
Building docker environment override files.
Unable to determine existing installation url.
Removing network docker_default
WARNING: Network docker_default not found.
Pulling web (bitwarden/web:1.22.0)...
1.22.0: Pulling from bitwarden/web
Digest: sha256:4b66b3ae5a2eec00b20c6125981378375c7e9c04eca5be30a96724b4a8c6bb25
Status: Image is up to date for bitwarden/web:1.22.0
Pulling attachments (bitwarden/attachments:1.16.0)...
1.16.0: Pulling from bitwarden/attachments
Digest: sha256:2b5de46153773ec8cba5ca99343dfb3813f1b9ee4e96760ff4a80851db2a5ba0
Status: Image is up to date for bitwarden/attachments:1.16.0
Pulling icons (bitwarden/icons:1.16.0)...
1.16.0: Pulling from bitwarden/icons
Digest: sha256:46bf6c5269ca13814783ec0230ced7d118633570fe800b7b58747b77d1ad6a65
Status: Image is up to date for bitwarden/icons:1.16.0
Pulling nginx (bitwarden/nginx:1.16.0)...
1.16.0: Pulling from bitwarden/nginx
Digest: sha256:049999a1beb189065b96721cee7014ba48a1c2d3a7fca2d24a0a31b96be73cfe
Status: Image is up to date for bitwarden/nginx:1.16.0
Pulling api (bitwarden/api:1.16.0)...
1.16.0: Pulling from bitwarden/api
Digest: sha256:c67e553b8a921dc48d69e32ae1e0977eb6e29508a7797155b5ddb69dc2c31f35
Status: Image is up to date for bitwarden/api:1.16.0
Pulling mssql (bitwarden/mssql:1.16.0)...
1.16.0: Pulling from bitwarden/mssql
Digest: sha256:619d0157d11f24acac720c970da2789f103b5ffa3169b1c7e67875083751e352
Status: Image is up to date for bitwarden/mssql:1.16.0
Pulling identity (bitwarden/identity:1.16.0)...
1.16.0: Pulling from bitwarden/identity
Digest: sha256:6f9c037290b1e03b38bb2d33de59b22af711d1d2b2b1769b2a33720f71c8cf70
Status: Image is up to date for bitwarden/identity:1.16.0
Creating network "docker_default" with the default driver
Creating bitwarden-identity ... done
Creating bitwarden-api ...
Creating bitwarden-icons ...
Creating bitwarden-web ...
Creating bitwarden-mssql ...
Creating bitwarden-nginx ...
Creating bitwarden-identity ...
Total reclaimed space: 0B
1.17.2: Pulling from bitwarden/setup
Digest: sha256:cf55e0288b6392ee9c35022089396c3c561773ddc963945f1afea8fce359757f
Status: Image is up to date for bitwarden/setup:1.17.2
```

**Comment by lyrixderaven** [comment]
Am I correct the current versions should be what's in the `bitwarden.sh` file?

```
COREVERSION="1.17.2"
WEBVERSION="1.24.1"
```

**Comment by kspearrin** [comment]
Yes those are right but the setup installer seems to be throwing errors. Have you modified your installation outside of the normal/supported install path?

**Comment by lyrixderaven** [comment]
I looked into it and yes, the issue was that the custom URL for my instance wasn't recognized by the updater for some reason. Removing the port from the url's in the `global.override.env` file did the trick, the instance updated again, and I was able to sync again. 

Sadly, there seems to be an Issue with the URI's of the credentials - they are all missing suddenly, and the web-interface looks slightly broken as well. Will look into it and open issues where appropriate. 

Thx for your help so far!
